### PR TITLE
TS settings

### PR DESCRIPTION
**Pull request by cjb**
@keybase/react-hackers

### CODE REVIEW

File: shared/settings/about-container.tsx [hunk 1]
```diff
@@ -28,6 +27,7 @@ const mapDispatchToProps = (dispatch, {navigateUp, navigateAppend}) => ({
 })
 
 const connectedHeaderHoc = Container.compose(
+  // @ts-ignore codemod-issue
   Container.connect<OwnProps, _, _, _, _>(
```
Comment (MarcoPolo) [inline shared/settings/about-container.tsx]: This should be fine if you remove the `<...>` i.e. it becomes `Container.connect(`

File: shared/settings/advanced/container.tsx [hunk 2]
```diff
@@ -25,7 +24,19 @@ const mapStateToProps = state => {
     lockdownModeEnabled: state.settings.lockdownModeEnabled,
     openAtLogin: state.config.openAtLogin,
     processorProfileInProgress: Constants.processorProfileInProgress(state),
-    setLockdownModeError: setLockdownModeError?.message ?? '',
+    // Auto generated from flowToTs. Please clean me!
```
Comment (MarcoPolo) [inline shared/settings/advanced/container.tsx]: It's unfortunate that TS doesn't support this null coalescing operator

File: shared/settings/feedback/container.desktop.tsx [hunk 3]
```diff
@@ -1,12 +1,11 @@
...
 import {namedConnect} from '../../util/container'
 import * as RouteTreeGen from '../../actions/route-tree-gen'
 import * as SettingsGen from '../../actions/settings-gen'
 import {anyWaiting} from '../../constants/waiting'
 import * as Constants from '../../constants/settings'
 
-type OwnProps = {||}
+type OwnProps = {}
 
 const mapStateToProps = state => ({
```
Comment (MarcoPolo) [inline shared/settings/feedback/container.desktop.tsx]: Oh, one more thing, when you remove the generics below, I think you'll have to type this function. i.e. `const mapStateToProps = (state: TypedState, ownProps: {}) ...`

**Comment by cjb** [comment]
@MarcoPolo Thanks!  That was easy.  Removed all the ts-ignores around connect() and namedConnect().

**Pull request merged**